<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="circleButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="polygonButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Polygon</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fillButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>4 Fill</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="colorButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill8Button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>8 Fill</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="widthBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1216</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_File"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/save.png</normaloff>:/rec/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/open.png</normaloff>:/rec/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/about.png</normaloff>:/rec/img/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/new.png</normaloff>:/rec/img/new.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
